plan/config.go

```diff
@@ -59,17 +61,25 @@ func ReadConfigFromEnviron() (*Config, error) {
 
 	var axes Axes
 	var behaviors Behaviors
+	filterMap := make(map[string][]Filter)
 	for _, e := range os.Environ() {
 		if strings.HasPrefix(e, axisKeyPrefix) {
 			axis := parseAxis(strings.TrimPrefix(e, axisKeyPrefix))
 			axes = append(axes, axis)
+		} else if strings.HasPrefix(e, behaviorSkipKeyPrefix) {
+			key, filter := parseSkipBehavior(strings.TrimPrefix(e, behaviorSkipKeyPrefix))
+			filterMap[key] = filter
```

Comment:
Okay if you look down below, we need to get the slice of filters per behavior when we iterate over behavior. The lookup becomes easier with map rather than having to do O( size of behavior * size of filter).